core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -57,7 +57,9 @@ class Analyzer(
   private var settings: Settings = _
   private var reporter: PresentationReporter = _
 
-  protected var scalaCompiler: RichCompilerControl = _
+  //Declared a more specific type in order to get access to RichPresentationCompiler `showRaw` method.
+  //If this is a bad decision, let me know and I will simply write wrapper method in `RichCompilerControl`
+  protected var scalaCompiler: RichCompilerControl with RichPresentationCompiler = _
```

Comment:
yeah, would be nice to have it in the RichCompilerControl so that we can keep track of what we're exporting.